app/src/main/java/fi/thl/koronahaavi/diagnosis/ShareConsentFragment.kt

```diff
@@ -1,6 +1,7 @@
 package fi.thl.koronahaavi.diagnosis
 
 import android.os.Bundle
+import androidx.hilt.navigation.fragment.hiltNavGraphViewModels
 import androidx.navigation.fragment.navArgs
```

Comment:
Classes with the hiltNavGraphViewModels change have unused import(s) (navGraphViewModels).

Reply:
thanks, fixed that and also found other places where hiltNavGraphViewModels should be used